.vib/magento/cypress/cypress/integration/magento_spec.js

```diff
@@ -0,0 +1,103 @@
+  cy.contains('Salable Quantity');
+  cy.get('#add_new_product-button').click();
+  cy.fixture('products.json').then((product) => {
+    cy.get('[name="product[name]"]').type(
+      `${product.newProduct.productName}.${random}`
+    );
+    cy.get('[name="product[price]"]').type(product.newProduct.price);
+  });
+  cy.get('#save-button').click();
+  cy.contains('You saved the product');
+  cy.get('#menu-magento-catalog-catalog').click();
+  cy.get('.item-catalog-products').click();
+  cy.fixture('products.json').then((product) => {
+    cy.contains(`${product.newProduct.productName}.${random}`);
+  });
+});
+
+it('allows user to create a customer account', () => {
+  cy.visit('/');
+  cy.contains('Create an Account').click();
+  cy.fixture('customers.json').then((customer) => {
+    cy.get('.field-name-firstname').type(
+      `${customer.newCustomer.firstName}.${random}`
+    );
+    cy.get('#lastname').type(`${customer.newCustomer.lastName}.${random}`);
+    cy.get('#email_address').type(`${random}.${customer.newCustomer.email}`);
+    cy.get('#password').type(customer.newCustomer.password);
+    cy.get('#password-confirmation').type(customer.newCustomer.password);
+  });
+  cy.contains('button', 'Create an Account').click();
+  cy.contains('Thank you for registering with Main Website Store.');
+  cy.fixture('customers.json').then((customer) => {
+    cy.get('.logged-in').contains(
+      `${customer.newCustomer.firstName}.${random}`
+    );
+  });
+});
+
+it('allows admin to add a discount', () => {
+  cy.login();
+  cy.get('#menu-magento-backend-marketing').click();
+  cy.get('.item-promo-catalog').click();
+  cy.get('#add').click({ force: true });
+  cy.fixture('discounts.json').then((discount) => {
+    cy.get('[name="name"]').type(`${discount.newDiscount.ruleName}.${random}`);
+    cy.get('.admin__actions-switch-text').click({ force: true });
+    cy.get('[class="admin__control-multiselect"]')
+      .first()
+      .select('Main Website');
+
+    cy.get('[class="admin__control-multiselect"]').last().select('General');
+    cy.get('#save').click();
+    cy.get('[name="discount_amount"]').type(discount.newDiscount.amount);
+    cy.get('#save').click();
+  });
+  cy.contains('You saved the rule');
+  cy.get('#apply_rules').click();
+  cy.contains('Updated rules applied.');
+});
+
+it('allows customer to subscribe to newsletter', () => {
+  cy.visit('/');
+  cy.fixture('customers.json').then((customer) => {
+    cy.get('#newsletter').type(`${random}.${customer.newCustomer.email}`);
+  });
+  cy.contains('Subscribe').click();
+  cy.contains('Thank you for your subscription.');
+  cy.login();
+  cy.get('#menu-magento-backend-marketing').click();
+  cy.get('.item-newsletter-subscriber').click();
+  cy.fixture('customers.json').then((customer) => {
+    cy.contains(`${random}.${customer.newCustomer.email}`);
+  });
+});
+
+it('allows customizing the home page', () => {
+  cy.login();
+  cy.get('#menu-magento-backend-content').click();
```

Comment:
Just to confirm, is there no option to navigate via `cy.visit()` to the intended page instead of going through UI navigation? Same for the other tests.

Reply:
I thought this as well and after trying I've concluded it's not possible. 
This is because the application appends a key during the navigation.
If I try navigating to the URL directly without the key, nothing happens. 

![Example](https://user-images.githubusercontent.com/47592680/178212664-553a4ac3-6eb7-4e80-9811-77d984271eb3.png)